packages/paste-core/components/button/stories/index.stories.tsx

```diff
@@ -60,17 +60,174 @@ const AllSizeOptions: React.FC<{variant: ButtonVariants}> = ({variant}) => {
 
 storiesOf('Components|Button', module)
   .addDecorator(withKnobs)
-  .add('Primary Button', () => <AllSizeOptions variant="primary" />)
-  .add('Secondary Button', () => <AllSizeOptions variant="secondary" />)
-  .add('Inverse Button', () => <AllSizeOptions variant="inverse" />)
-  .add('Destructive Button', () => <AllSizeOptions variant="destructive" />)
-  .add('Destructive Secondary Button', () => <AllSizeOptions variant="destructive_secondary" />)
-  .add('Destructive Link Button', () => <AllSizeOptions variant="destructive_link" />)
-  .add('Link Button', () => <AllSizeOptions variant="link" />)
-  .add('Inverse Link Button', () => <AllSizeOptions variant="inverse_link" />)
+  .add('Primary Button', () => (
+    <>
+      <AllSizeOptions variant="primary" />
+      <Box padding="space30">
+        <Stack orientation="vertical" spacing="space40">
```

Comment:
I think you can just pop these into the `allButtons` array, pass the variant down and not have to touch the individual stories in this case